js/index.js

```diff
@@ -1 +1,36 @@
-// Your code goes here
+let navLinks = document.querySelectorAll('a');
```

Comment:
Try and use a more specific selector such as `nav a` so that if more `a` elements are added to the page it won't cause unexpected behaviour.